roles/openshift_excluder/tasks/status.yml

```diff
@@ -20,13 +20,15 @@
 - name: Update to latest excluder packages
   package:
     name: "{{ openshift.common.service_type }}-excluder"
+    state: latest
```

Comment:
Unfortunately earlier versions of the excluders lacked a status function so I'd really like to ensure they're up to date so that we can rely on the status function. I've also just realized that we're not actually calling the status function below so we should fix that.